academic_observatory_workflows/workflows/tests/test_crossref_metadata_telescope.py

```diff
@@ -18,15 +18,21 @@
 import os
 from datetime import datetime
 from unittest.mock import patch
-import requests
 
 import httpretty
 import pendulum
+import requests
 from airflow.exceptions import AirflowException
 from airflow.models import Connection
 from airflow.utils.state import State
 from click.testing import CliRunner
 from natsort import natsorted
+from observatory.api.client.model.dataset import Dataset
```

Comment:
Cosmetic question - why are the imports ordered like this?

Reply:
You're right it was odd. I'm not sure how they ended up like that! I use PyCharm to order them and removed unused imports, but not sure what it did.